src/main/java/net/isaac/chess/Board.java

```diff
@@ -15,17 +15,34 @@
     private List<Piece> blackPieces = new ArrayList<>();
 
     public void initialize(){
+        initializeSpecialPieces(Piece.Color.BLACK,0);
+        initializePawns(Piece.Color.BLACK,1);
+        initializePawns(Piece.Color.WHITE,MAX_IDX-1);
+        initializeSpecialPieces(Piece.Color.WHITE, MAX_IDX);
```

Comment:
코드 일관성을 위해서 꼭 리포맷을 해주세요.

Reply:
```suggestion
        initializePawns(BLACK, 1);
        initializePawns(WHITE, MAX_IDX - 1);
        
        initializeSpecialPieces(BLACK, 0);
        initializeSpecialPieces(WHITE, MAX_IDX);
```
저는 색단위보다는 하는 행위에 초점을 맞춰봤어요. 그리고 너무 장황하게 코딩하기보다는 직관적으로 읽히게끔 코딩하는 방법도 생각해보세요.

Reply:
또, 1이랑 0은 하드코딩 되어있는데, 중복이 되어있습니다. 한 번 수정해볼까요?